src/plugins/region_map/server/routes/opensearch.ts

```diff
@@ -14,20 +14,28 @@ export function registerGeospatialRoutes(router: IRouter) {
         body: schema.object({
           index: schema.string(),
         }),
+        query: schema.maybe(schema.object({}, { unknowns: 'allow' })),
```

Comment:
why we use `{ unknowns: 'allow' }` here? isn't dataSourceId the only parameter allowed?